client/src/components/Footer.js

```diff
@@ -15,28 +15,42 @@ const Footer = styled.footer`
 `;
 
 const List = styled.ul`
+  width: 100%;
   display: flex;
-  felx-direction: row;
+  flex-direction: row;
   justify-content: space-between;
 
   @media (max-width: 900px) {
     flex-direction: column;
+    justify-content: space-evenly; /* I tried to use this to create some space when
+    screen size is small but it didn't work..*/
```

Comment:
I actually never used `space-evenly` I think, so not sure

Reply:
![Reversim_Summit_2019](https://user-images.githubusercontent.com/1729783/63776390-fda69a00-c8e9-11e9-9b64-345db6638e9a.png)

Reply:
Defenitly not doing what I want. Question is, how should I fix it?